lib/models/BucketInfo.ts

```diff
@@ -285,6 +288,9 @@ export default class BucketInfo {
             tags = [] as BucketTag[];
         }
         assert.strictEqual(areTagsValid(tags), true);
+        if (quota) {
+            assert.strictEqual(typeof quota, 'number');
+        }
```

Comment:
maybe also if the number if negative, we should reject it?

Reply:
done here : https://github.com/scality/Arsenal/pull/2234/commits/a5541f28bc6eb5fe46e0ac5e658a2be100057e15#diff-4ee3cd5de98073dadb72b27f034f4979480c04388cc4f4538f4895373527f174R291-R293